mirage/dune

```diff
@@ -1,4 +1,4 @@
 (library
  (name capnp_rpc_mirage)
  (public_name capnp-rpc-mirage)
- (libraries capnp-rpc-lwt capnp-rpc-net capnp-rpc fmt logs mirage-stack dns-client.mirage))
```

Comment:
Shouldn't this be depending on tcpip instead then?

Reply:
I works without it, my guess is that tcpip is a transitive dependency.

Reply:
I guess dns-client brings it in. But it would be clearer to be explicit here.

Reply:
Done!